### PR TITLE
Submit form added

### PR DESCRIPTION
**Pull request by tatjana-kolosek**
@kristina-kolosek Can you please take a look at my work?

### CODE REVIEW

File: src/Checkbox/Checkbox.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import '../style.css';
+
+const Checkbox = (props) => {
+  return (
+      <div>
+        <h6>I agree with terms and conditions</h6>
+        <input className="form-input"
+               type="checkbox"
```
Comment (kristina-kolosek) [inline src/Checkbox/Checkbox.js:9]: Make this checkbox controlled

File: src/Form/Form.js [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+      email: '',
+      first: '',
+      last: '',
+      password: '',
+      checked: true
+    }
+  }
+
+  handleEmailChange = (event) => {
+    this.setState({email: event.target.value});
```
Comment (kristina-kolosek) [inline src/Form/Form.js]: Let's have one function handling the values of inputs, there's no need for 4 of them

File: src/Form/Form.js [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+  handleLastChange = (event) => {
+    this.setState({last: event.target.value});
+  }
+
+  handlePassChange = (event) => {
+    this.setState({password: event.target.value});
+  }
+
+  handleCheckChange = (event) => {
+    if (this.state.checked) {
```
Comment (kristina-kolosek) [inline src/Form/Form.js]: Pretty weird way to handle this (and creative, it works indeed). Let's make a checkbox that is properly controlled instead :D

**Review by kristina-kolosek — CHANGES_REQUESTED** [review]
Looking pretty good, just please make these small tweaks

**Comment by tatjana-kolosek** [comment]
@kristina-kolosek Thank you, I will make the adjustments

**Review by kristina-kolosek — COMMENTED** [review]
Good job on fixing these!

**Pull request closed without merging**